apps/site/components/Downloads/Release/ReleaseCodeBox.tsx

```diff
@@ -1,12 +1,12 @@
 'use client';
 
+import AlertBox from '@node-core/ui-components/Common/AlertBox';
+import Skeleton from '@node-core/ui-components/Common/Skeleton';
```

Comment:
Can we make an eslint import rule, so that @node-core imports get sepoarated from rest of imports?